ytfzf

```diff
@@ -802,13 +804,46 @@ scrape_pt () {
 		thumbs: .thumbnailUrl
 	    }
 	]')
-     export videos_json
+     printf "%s" "$videos_json" > "$tmp_video_json_file"
      videos_data="$(get_video_data "$videos_json")"
      function_exists "on_video_data_gotten" && on_video_data_gotten "$videos_data" "$videos_json" "$pt_json"
      [ $show_thumbnails -eq 1 ] && download_thumbnails "$videos_json"
      unset pt_json
 }
 
+scrape_odysee () {
+    odysee_json=$(
+	curl \
+	    -s "https://lighthouse.lbry.com/search" \
+	    -G --data-urlencode "s=$*" \
+	    -G --data-urlencode "include=channel,title,thumbnail_url,duration,cq_created_at,content_type,description"
+    )
+    [ -z "$odysee_json" ] && print_error "ERROR[#01]: Couldn't curl website. Please check your network and try again.\n" && exit 1
+
+    videos_json=$(printf "%s" "$odysee_json" |\
+	jq '[ .[] | select(.content_type == "video/mp4" or .content_type == "video/m4v" or .content_type == "video/webm") | 
```

Comment:
remove this not needed anymore
`| select(.content_type == "video/mp4" or .content_type == "video/m4v" or .content_type == "video/webm")`